rpcd/playbooks/roles/rpc_pre_upgrade/tasks/backup.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Create upgrade backup directory
+  file:
+    path: "{{ backup_dir }}"
+    state: directory
+
+- name: Backup the /etc/openstack_deploy directory
+  command: "tar czf {{ backup_dir }}/openstack_deploy-{{ datetime_stamp }}.tar.gz /etc/openstack_deploy"
+  tags:
+    - skip_ansible_lint
```

Comment:
This tag doesn't seem very informative. Why has this task and one in log_rotation.yml been given this tag?